libs/util/nestjs/kafka/src/pacemaker/pacemaker.service.ts

```diff
@@ -0,0 +1,33 @@
+import { KafkaContext } from "@nestjs/microservices";
+import { promisify } from "util";
+
+export class KafkaPacemaker {
+  static async wrapLongRunningMethod<T>(
+    kafkaContext: KafkaContext,
+    fn: () => Promise<T>,
```

Comment:
could this accept any functions like 
```suggestion
    fn: (...args: unknown[]) => Promise<T>,
```